.github/workflows/unit-test.yml

```diff
@@ -11,7 +11,7 @@ jobs:
       matrix:
         node: ["14", "16", "18.18.2"]
         include:
-            - node: 14
+            - node: 16
```

Comment:
Note to reviewers: This was discussion in the OTel JS SIG this week as a fix for the code-coverage check failures. Testing coverage only with node v14 artificially results in coverage looking poor for a unit-test run that skips all or most tests for the (now old) node v14.